tools/docker/docker-compose-dev.yaml

```diff
@@ -17,7 +17,7 @@ x-environment:
 services:
   podman:
     user: "1000"
-    image: quay.io/containers/podman:${EDA_PODMAN_VERSION:-v4.5}
+    image: quay.io/containers/podman:${EDA_PODMAN_VERSION:-v4.6}
```

Comment:
I think it will be better just use v4, to avoid the same problem

Reply:
Works for me.